static-build/test/static-build/exports.test.lua

```diff
@@ -128,10 +128,7 @@ local check_symbols = {
     'HMAC_Final',
 
     'ZSTD_compress',
-    'ZSTD_decompress',
-    'ZSTD_free',
-    'ZSTD_malloc',
```

Comment:
> Removed obvious exporting functions from static-build test:

I don't understand what you meant here. Obvious?

Reply:
Now it is stated as 'outdated' and it is so. However, what is more important, they never were public symbols and should not be tested for presence in the executable.

Please, add this information into the commit message.

Reply:
Sure, commit message updated with the following:

Found that some checking in static-build test exporting symbols like:
```    
      ZSTD_free
      ZSTD_malloc
```    
never were public symbols and should not be tested for presence in the
Tarantool executable, but also these symbols currently outdated and
broke the testing. To avoid of it these symbols removed from test.